vendor/vendor.json

```diff
@@ -383,7 +383,7 @@
 			"revisionTime": "2016-12-29T22:47:41Z"
 		},
 		{
-			"checksumSHA1": "oBjdFvDUZpioGsB6URqe16hDQUg=",
```

Comment:
any idea why these checksums changed?

Reply:
Not exactly sure. I'll remove it from the patch.